docs/support.rst

```diff
@@ -0,0 +1,316 @@
+
+Chaos V-Ray
+===========
+
+`V-Ray <https://www.chaos.com>`_ is a 3D path-traced renderer
+that supports USD and has a Hydra delegate.
+
+`V-Ray Documentation <https://www.chaos.com/blog/getting-started-with-usd-and-hydra-in-v-ray>`_
+
+Gaffer
+======
+
+`Gaffer <https://www.gafferhq.org>`_ is an `open source <https://github.com/GafferHQ/gaffer>`_ , node based application for lighting and compositing.
+
+Golaem
+======
+
+`Golaem <https://golaem.com>`_ is a crowd simulation program with an `open source, USD plugin <https://github.com/Golaem/GolaemForUSD>`_
+that allows viewing Golaem crowd caches within USD.
+
+Guerilla Render
+===============
+
+`Guerilla Render <http://guerillarender.com>`_ is a 3D renderer.
+
+Foundry
+=======
+
+The Foundry are a developer of 3D Content Creation, Lighting and Compositing software.
+
+Katana
+------
+
+`Katana <https://www.foundry.com/products/katana>`_ is  look development and lighting package.
+
+Katana has `open source USD plugins <https://learn.foundry.com/katana/dev-guide/Plugins/KatanaUSDPlugins/index.html>`_ as
+well as a support for `Hydra render delegates <https://learn.foundry.com/katana/dev-guide/Plugins/HydraRenderDelegates/index.html>`_.
+
+Mari
+----
+
+`Mari <https://github.com/Golaem/GolaemForUSD>`_ is a 3D texture painting application.
+
+Modo
+----
+
+`Modo <https://www.foundry.com/products/modo>`_ is a general purpose 3D package.
+
+Nuke
+----
+
+`Nuke <https://www.foundry.com/products/nuke-family/nuke>`_ is a compositing package with support for both USD and Hydra.
+
+`Nuke documentation <https://learn.foundry.com/nuke/content/comp_environment/3d_compositing/usd.html>`_
+
+
+Isotropix Clarisse
+==================
+
+`Clarisse <https://www.isotropix.com/products/clarisse/ifx>`_ is a 3D application for look development, lighting and rendering.
+
+Maxon
+=====
+
+Maxon are a developer of 3D content creation, rendering and motion graphics software.
+
+Cinema 4D
+---------
+
+`Cinema 4D <https://www.maxon.net/en/cinema-4d>`_ is a general purpose 3D package.
+
+Redshift
+--------
+
+`Redshift <https://www.maxon.net/en/redshift>`_ is a GPU-accelerated path-traced renderer.
+
+Redshift supports USD and is also available as a Hydra delegate for rendering integration.
+
+ZBrush
+------
+
+`ZBrush <https://www.maxon.net/en/zbrush>`_ is a 3D sculpting and painting application.
+
+`ZBrush USD documenation <http://docs.pixologic.com/user-guide/zbrush-other-programs/usd-format/>`_
+
+
+NVIDIA Omniverse
+=================
+
+`NVIDIA Omniverse <https://www.nvidia.com/en-us/omniverse/>`_ is a platform for creating and operating metaverse applications.
+It is based on USD.
+
+`NVIDIA USD Documentation <https://www.nvidia.com/en-us/omniverse/usd/>`_
+
+Omniverse also adds their own USD support to the following applications:
+
+* 3ds Max
+* Archicad
+* Character Creator
+* Creo
+* iClone
+* ParaViw
```

Comment:
They also got a maya connector !
```suggestion
* iClone
* Maya
* ParaViw
```

Reply:
Ah good catch. I found their list of connectors and linked that instead. I then pruned the list to just the items that aren't listed elsewhere so as to keep the list small but still show that there is a way to get these applications into a persons pipeline.